继续挑战

---
### 第15题地址[uzi.html](http://www.pythonchallenge.com/pc/return/uzi.html)
* <img src="http://huge:file@www.pythonchallenge.com/pc/return/screen15.jpg" alt="screen15.jpg" width="30%" height="30%">
* 网页标题是`whom?`，题目内容为空，[源码](view-source:http://www.pythonchallenge.com/pc/return/uzi.html)中有两行隐藏信息：
> <!-- he ain't the youngest, he is the second --\><br>
> <!-- todo: buy flowers for tomorrow --\>

这个题目只有图片，感觉算是比较明显的。<br>
图片上是一张**1??6年1月**的日历，上面圈出来一个**26**号，隐约看得清楚是**周一**。结合标题就可以知道，就是找出出生在这一天的名人是谁。

### 慢着！！
正当我要开始写代码的时候，突然意识到，**1??6年**总共有**100**个，**1月26号**是**周一**的年份肯定不少，这里面肯定还有坑。<br>
再仔细看那张日历，发现右下角还有两张小日历，模模糊糊应该是前后两个月的，是上一年12月和这一年**2月**的。
### 等等！！2月？？
我瞪大了眼睛数了一下，这个**2月**是有**29**天的，也就是说，这个**1??6年**是个闰年！！果然有坑！

---

好吧，可以开始写了：

In [1]:
import calendar

for year in range(1996, 1000, -10):
    if calendar.isleap(year) and calendar.weekday(year, 1, 26) == calendar.MONDAY:
        print(year)

1976
1756
1576
1356
1176


找出了**5个**年份符合条件！<br>
再根据隐藏信息，*他是第二年轻的*，所以应该是**1756年**，搜索**1756年1月26日**出生的名人，结果找到一个
> 1月27日——沃尔夫冈·阿玛多伊斯·莫扎特（Wolfgang Amadeus Mozart），奥地利作曲家（逝世1791年）<br>
> ###### From [wikipedia.org](https://zh.wikipedia.org/wiki/1756%E5%B9%B4)

呃，是**1月27日**。。。再看到隐藏信息第二条，也凑合用上了吧，就是他了，**Mozart**！<br>
把地址改为[mozart.html](http://www.pythonchallenge.com/pc/return/mozart.html)，果然来到了下一题。

### 总结：题目实现很简单，就是图片要看仔细，很多坑！
###### 本题代码地址[15_uzi.ipynb](https://github.com/StevenPZChan/pythonchallenge/blob/notebook/nbfiles/15_uzi.ipynb)